content/spin/v2/key-value-store-tutorial.md

```diff
@@ -13,12 +13,10 @@ url = "https://github.com/fermyon/developer/blob/main/content/spin/v2/key-value-
   - [The Spin TOML File](#the-spin-toml-file)
 - [Write Code to Save and Load Data](#write-code-to-save-and-load-data)
   - [The Spin SDK Version](#the-spin-sdk-version)
-- [Quick Overview - Video](#quick-overview---video)
-- [Source Code](#source-code)
+  - [Source Code](#source-code)
 - [Building and Deploying Your Spin Application](#building-and-deploying-your-spin-application)
 - [Storing and Retrieving Data From Your Default Key/Value Store](#storing-and-retrieving-data-from-your-default-keyvalue-store)
 - [Conclusion](#conclusion)
-- [Next Steps](#next-steps)
 
 ## Key Value Store With Spin Applications
 
```

Comment:
Can't comment on next para but it links to a blog on Spin 0.9!

Reply:
I don't see that in here anymore?